requirements.txt

```diff
@@ -2,6 +2,6 @@ datajoint<=0.12.9
 nnfabrik
 torch
 scipy>=1.7.0,<=1.12.0
-numpy>=1.22.0,<=1.26.4
+numpy>=1.21.0,<=1.26.4
```

Comment:
Why was it previously less loose, do we know?

Reply:
I am not aware that we would know. @PPierzc do you have a hunch?